backend/kale/common/snapshotutils.py

```diff
@@ -0,0 +1,119 @@
+#  Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
The License header is missing the first line: `# Copyright 2020 The Kale Authors`

Reply:
And of course you can file a PR to be added to the `AUTHORS` file